redis.conf

```diff
@@ -820,6 +820,19 @@ acllog-max-len 128
 #
 # requirepass foobared
 
+# New users are initialized with restrictive permissions by default, via the
+# equivalent of this ACL rule 'off -@all'. Starting with Redis 6.2, it is
+# possible to manage access via ACL for Pub/Sub channels as well. To ensure
+# backwards compatibility, new users are granted the 'allchannels' permission
+# by default.
+#
+# Future compatibility note: it is very likely that in a future version of Redis
+# the directive's default of 'allchannels' will be changed to 'resetchannels' in
+# order to provide better out-of-the-box Pub/Sub security. Therefore, it is
+# recommended that you explicitly define Pub/Sub permissions for all user rather
+# then rely on the implicit defaults.
+acl-pubsub-default allchannels
```

Comment:
maybe we better put that last line (default behavior) in a comment, this way we can later distinguish between people who explicitly set it, and ones that left it unchanged?

p.s. i find it hard to locate the various options for this config in the text above.
maybe add an explicit list of options (like the one for `appendfsync` or `maxmemory-policy`)

Reply:
Done.